src/DynamoCoreWpf/Properties/AssemblyInfo.cs

```diff
@@ -30,3 +30,4 @@
 [assembly: InternalsVisibleTo("DynamoMSOfficeTests")]
 [assembly: InternalsVisibleTo("CoreNodeModelsWpf")]
 [assembly: InternalsVisibleTo("DynamoSandbox")]
+[assembly: InternalsVisibleTo("MutationTestInfrastructure")]
```

Comment:
@pboyer I'm not fantastically happy about using this mechanism -- but this is a test tool so it does have some plausibility.

We could open it up as an API via the extension but I'm not sure we have the bandwidth to add the APIs with the design attention they'd need right now. What are you thoughts here?